05-how-to-contribute-to-open-source.md

```diff
@@ -25,139 +29,293 @@ OpenSauced is a powerful tool for finding open source projects to contribute to.
+- A screenshot or screen recording when you make a UI change
 
-#### Step-by-Step Guide
+#### Resolving Merge Conflicts Immediately
 
-1. Open VS Code: Launch Visual Studio Code and open the repository where the merge conflict exists.
-2. Check for Conflicts: If you've just performed a pull or merge operation and a conflict occurs, VS Code will automatically detect it.
-You will see a notification in the source control tab or sidebar.
-3. View Conflicted Files: Click on the conflicted file in the source control tab. VS Code will display a "Merge Conflict" section in the file.
-4. Resolve Conflicts: Inside the "Merge Conflict" section, you'll see the conflicting changes separated by the conflict markers (<<<<<<<, =======, >>>>>>). From there, decide which changes to keep or combine.
-5. Save the file: After resolving the conflicts, save the file.
-6. Stage the Resolved File: In the Source Control tab, you should see options to stage the resolved file. Click on the "Mark as Resolved" or similar button.
-7. Commit the Changes: Once all conflicts are resolved, commit your changes using VS Code's built-in source control features.
-8. Push Your Changes: After committing, push the changes to the remote repository to complete the merge process.
-9. Verify the Merge: Verify that the merge conflict has been successfully resolved by checking the remote repository or using Git tools.
+When you notice merge conflicts in your PR or if a maintainer asks you to resolve merge conflicts, fix them immediately.
+
+### Merge Conflicts in the Guest Book Repository
+
+Since the primary purpose of the guest book is to add your name to `.all-contributorsrc` and the `README.md` files, there is a high chance that you will encounter merge conflicts.
+
+The conflicts happen when maintainers have merged pull requests before yours while you're working on your changes or waiting for your pull request to be reviewed. And you need to resolve them before your pull request can be merged.
+
+#### Resolving Merge Conflicts
+
+Before resolving merge conflicts, you must first [update your branches](#updating-branches). Then, follow these steps:
+
+1. In the `.all-contributorsrc` file:
+
+   - Click the "Accept Both Change" option on the top of your workspace in VS Code.
+   - Move your profile details to the end of the contributors' array and fix anything necessary.
+
+2. In the `README.md` file:
+
+   - Click the "Accept Incoming Change" option on the top of your workspace in VS Code for each conflict in this file.
+
+3. Run `npm run contributors:generate`.
+
+   You will now see that the all contributors badge has been incremented, and your profile is generated at the end of the contributors' list in the `README.md` file.
+
+4. Add and commit your changes.
+
+   ```bash
+   git commit -am "Resolve merge conflicts"
+   ```
+
+5. Push your commits to your remote branch.
+
+   ```bash
+   git push
+   ```
 
 ## Onboarding in a New Project
```

Comment:
This is just a thought. It might make more sense to move this section somewhere on the top, before "Getting Started With Contributing" section. What do you think?

Reply:
I agree @adiati98 . It will show beginners the importance of being apart of a community when it comes to open source projects.